integrations/woopra/test/index.test.js

```diff
@@ -38,14 +38,11 @@ describe('Woopra', function() {
         .option('cookieName', 'wooTracker')
         .option('cookieDomain', null)
         .option('cookiePath', '/')
-        .option('ping', true)
-        .option('pingInterval', 12000)
         .option('idleTimeout', 300000)
         .option('downloadTracking', true)
         .option('outgoingTracking', true)
+        .option('clickTracking', true)
```

Comment:
I don't actually see what the default value for `clickTracking` is on the woopra docs. If it's already `true` then this is totally fine.

Reply:
Good point. `clickTracking` is not turned on by default in the tracker for retroactive reasons. But we would like to be turned on for every new installation. Is it possible to turn it off for existing installations and automatically turn it on for new installations?

Reply:
Sorry for the delayed response - it looks like it isn't possible to turn it off for existing installs and have it defaulted on for new installations. We would need to set it as `false` by default so that it doesn't change existing behavior.